clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -48,6 +58,21 @@ class ElasticsearchIndex[CI <: ClioIndex](
     s"$rootDir$dir/"
   }
 
+  /**
+    * The ID of a json record in elasticsearch. By default, each record's ID consists
+    * of the concatenated values of the Key fields.
+    * Additional fields can be added to the ID using the ElasticsearchIndex constructor.
+    */
+  def getId(json: Json): String = {
+    val keyFields = FieldMapper[clioIndex.KeyType].fields.keys
+      .map(snakeCaseTransformation)
+    val foo =
+      (keyFields ++ additionalIdFields).toSeq.sorted
```

Comment:
Using `sorted` will change the ordering of the ID pieces for ubams, which we weren't planning on rebuilding as part of this... will poll the group about it.

Reply:
Not sure what the motivation of using `sorted` here is. Is the intent to alpha sort all key fields for each ID? I think we'd want to have the key fields in the same order for all IDs in an index.